src/actions/start-irma-session.js

```diff
@@ -1,17 +1,33 @@
 const diva = require('diva-irma-js');
 const moment = require('moment');
 
+function startIssueSession(credentialType, sessionId) {
+  switch (credentialType) {
+    case 'EAN':
+      throw new Error(`Error starting ${sessionId}. Not supported yet!`);
+    default:
+      return diva.startIssueSession([{
+        credential: 'irma-demo.MijnOverheid.address',
+        validity: moment().add(6, 'months').unix(),
+        attributes: {
+          country: 'The Netherlands2',
```

Comment:
Maybe use a different, very common address like The Hague where parliament resides.